pyro_risks/datasets/fwi.py

```diff
@@ -225,7 +226,7 @@ def create_departement_df(day_data):
 class GwisFwi(pd.DataFrame):
     """GWIS FWI dataframe (8 km resolution) on French territory based on 2019-2020 data."""
 
-    def __init__(self, days_list=["20190101"]):
+    def __init__(self, days_list: List[str] = ["20190101"]) -> None:
```

Comment:
See codacity: 
https://app.codacy.com/gh/pyronear/pyro-risks/file/55703095880/issues/source?bid=20424053&fileBranchId=23145733#l228

```suggestion
    def __init__(self, days_list: Optional[List[str]] = None) -> None:
        day_list = ["20190101"] if day_list is None else day_list
```